### ISSUE TITLE
Suggestion: Folders on Desktop

### ISSUE THREAD

**Issue by playful1510** [NeoApplications/Neo-Launcher#204]
I'm coming from Nova and I love this launcher. Right now the only reason that I haven't switched over completely is the lack of support for having my folders on the desktop for easy access. My suggestion then is to bring this feature over to Neo.

**Comment by FlareHeart** [comment]
I have folders on my desktop (home screen). Are you referring to a specific kind of folder? I just dragged some shortcuts onto each other to create folders and it worked fine for me.

**Comment by playful1510** [comment]
How did you manage to do that? I tried to do the same method I'd use to move apps onto the desktop, e.g. press-hold-drag, and nothing happens. I asked about this in the telegram group and the dev said that this isn't something that's implemented.

To answer your question, I'm referring to any folder made in the app drawer.

**Comment by FlareHeart** [comment]
I just touch-hold and drag one icon onto another. Right on top of it and it created a folder. I have 5 folders on my home screen with 6 apps per folder. Works just the same as in Nova for me.

**Comment by playful1510** [comment]
Oh I see how you did that now. I'm not talking about making them on the home screen itself, but rather, making the folders in the app drawer and then pulling them, as is, onto the home screen.

**Comment by FlareHeart** [comment]
Ahhh I see. Sorry I thought you meant just regular folders.

**Comment by playful1510** [comment]
Yeah, that's how I've done it in Nova, since when you change the folder configuration in the app drawer, it automatically updates the folder on the home screen so you don't have two different versions of the same folder.

**Comment by machiav3lli** [comment]
@playful1510 but what's the use of such duplicates?

**Comment by playful1510** [comment]
So that the app drawer is organized properly, and to have the same organization on the home screen, I suppose? I don't like to have a bunch of homescreens to scroll through to find an app, when I could just have as many things as possible in folders and condense it down to a single screen.

**Comment by Eschguy** [comment]
This is one of the biggest features I'm waiting for.

**Comment by tdadvocate** [comment]
This is the second most important missing feature for me that's holding me back from switching over from Nova. This and having both folders & tabs in the app drawer at the same time rather than one or the other. Then it's perfect.

**Comment by butjer** [comment]
Very important feature for mee too.
And to edit the icon for these folders.
